src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/Commands/SetAzureRmDataLakeAnalyticsAccount.cs

```diff
@@ -63,8 +63,34 @@ public class SetAzureDataLakeAnalyticsAccount : DataLakeAnalyticsCmdletBase
         [ValidateNotNull]
         public TierType? Tier { get; set; }
 
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 4, Mandatory = false,
```

Comment:
@begoldsm is this position attribute needed? If so, it looks like it should be 3 instead of 4